components/compositing/compositor.rs

```diff
@@ -396,6 +399,7 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             last_composite_time: 0,
             ready_to_save_state: ReadyState::Unknown,
             scroll_in_progress: false,
+            in_scroll_transaction: None,
```

Comment:
Should `on_scroll_start_window_event` and `on_scroll_end_window_event` also modify this field?